### DISCUSSION TITLE
[APyB] Como gastar o dinheiro do evento?

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#158] [Geral]
Prezadas pessoas organizadoras de eventos,

Estamos bem felizes que esse ano todos os eventos regionais foram realizados ou estão em vias de. Em breve nos encontraremos na Python Brasil. Gostaríamos de trazer algumas informações sobre como deve ser utilizado o dinheiro que entra no evento.

Sabe aquela história que a organização gasta o dinheiro como quiser? Apesar da organização ter autonomia sobre a escolha da grade e dos fornecedores do evento, os gastos devem estar alinhados com a atividade-fim da Associação. A APyB é uma associação civil de direito privado, sem fins lucrativos. Sendo assim, sob a perspectiva contábil a Associação é regida pela Lei 6.404/76 e há como guia o Manual de Contabilidade Societária, aplicável para todas as sociedades de acordo com as normas internacionais e do Código de Processo Civil.

Na prática isso quer dizer que o gasto deve estar relacionado com a atividade da Associação (CNAE primário 9499-5/00, CNAE secundária 9493-6/0), alguns exemplos:

| **PODE** | **NÃO PODE** |
| --| --|
| Alimentação (comida e bebida não alcoólica) | Bebida alcoólica |
| Apresentação cultural | Serviços de modificação corporal (piercing, tatuagem, alargadores e afins) |
| Aluguel de espaço | Aquisição de qualquer equipamento, patrimônio e material permanente |
| Serviço de limpeza | |
| Serviço de áudio visual | |
| Brindes | |

O dinheiro excedente do caixa de todos os eventos que usam a estrutura da APyB é redirecionado da seguinte forma: 50% destinado para o [Fundo para Inclusão](https://apyb.python.org.br/apyb/fundo-inclus%C3%A3o) e 50% para manter a estrutura utilizada pelos eventos.

O [Fundo para Inclusão](https://apyb.python.org.br/apyb/fundo-inclus%C3%A3o) é uma iniciativa aprovada durante a [Assembleia Geral de 2021](https://docs.google.com/document/d/1lz56OI3TLKvJdfjyC6RARnJZwZwYHmoqvX8GCrCMm1M/edit?usp=sharing), sendo possível pelos esforços da organização da Python Brasil daquele ano que, devido ao contexto econômico do alto valor arrecadado dos patrocínios, a comunidade viu a possibilidade de usar esse valor arrecadado para criação de um fundo para dar suporte à iniciativa cujo objetivo seja promover ação de diversidade e inclusão nas edições presenciais do evento. O Fundo serve para garantir o acesso de grupos historicamente sub-representados nas edições presenciais do evento, já que o evento que teve excedente esgotou as formas de utilizar o seu caixa para garantir o fortalecimento regional da comunidade.

A saber, esta foi a [ação promovida para a Python Brasil 2022](https://2022.pythonbrasil.org.br/programa-auxilio-financeiro), o qual a prestação de contas foi apresentada nesta [palestra relâmpago](https://www.youtube.com/live/6SShnyYTCv0?si=eKITpfvLNnWIdSSa&t=288).

Entendemos que organizar uma conferência de tecnologia desse porte é desafiador, mas temos algumas dicas adicionais:

- Leia o [Manual da Big Kahuna](https://apyb.python.org.br/associados/guias/como-organizar-eventos/)
- Leia o [Estatuto da APyB](https://github.com/apyb/estatuto)
- Observe os [repositórios](https://github.com/pythonbrasil/) das organizações das edições anteriores
- Busque, ainda nos primeiros meses de organização, pessoas com perfil de liderança com ótimas habilidades interpessoais para formar [Código de Conduta](https://apyb.python.org.br/pythonbrasil/cdc/)
- Em caso de dúvida, abra uma issue no [Github Discussions](https://github.com/apyb/comunidade/discussions) da comunidade

Boa organização de evento para todas!!

@apyb/diretoria Gestão 2024/2025

**Comment by cecivieira** [comment]
cc @anapaulamendes @Emilly-horta @renan-asantos @biancarosa @ericof

**Comment by naanadr** [comment]
cc @vbuxbaum

**Comment by naanadr** [comment]
@MateusSantosMeuBem quem é a pessoa BigKahuna da próxima PyN?

**Reply by meu-bem** [reply]
Oi, pessoal! Tudo bem com vocês?
A pessoa Big Kahuna da Python Norte 2025 será o @jailsonpj!

**Reply by naanadr** [reply]
Obrigada @MateusSantosMeuBem ! Recomendo que vcs leiam o informe desse Discussion e caso tenham alguma dúvida, não deixem de perguntar.

**Comment by naanadr** [comment]
@talitarossari quem é a pessoa BigKahuna da próxima PySul?

**Reply by talitarossari** [reply]
roieee, o kahuna da PySul 2025 é o @jbiason <3

**Reply by naanadr** [reply]
Obrigada @talitarossari , recomendo que vcs leiam o informe desse Discussion e caso tenham alguma dúvida, não deixem de perguntar.

**Comment by biancarosa** [comment]
Oi @cecivieira e @naanadr! Super válido.

Não pergunto isso necessariamente para a Python Brasil desse ano, portanto não há urgência em responder, mas muito mais como esclarecimento do que é um material permanente. 

Alguns gastos como canetas, cronômetros, tesouras, máquina de impressão em etiquetas etc, essas coisas de papelaria de última hora e algumas coisas que são diretamente ligadas a experiência do evento podem ser julgadas ou necessárias ou interessantes pela pessoa organizadora. Como vocês interpretam esses gastos? O que é considerado um material permanente? Algumas dessas coisas não poderiam ser de fato devolvidas para a apyb se eventualmente criássemos a estrutura e regras para tais? Não sei se temos essa estrutura hoje, mas vocês enxergam que na situação em que essa estrutura fosse criada, isso poderia ser revisitado?

Abraços e obrigada pelo esclarecimento e paciência ❤️

**Reply by cecivieira** [reply]
@biancarosa não temos estrutura física da APyB para armazenar tais materiais. Então, caso seja necessário comprar material de papelaria, os mesmos podem ficar sob o poder da organização local, tendo em vista o baixo tempo de vida útil desses materiais.

O que queremos dizer me materiais permanentes são aqueles de alto valor, que pode ser considerado também como patrimônio, por exemplo: eletrônicos e periféricos.

**Comment by vbuxbaum** [comment]
Ei @cecivieira ! Tudo bem?

No [guia de organização de eventos](https://apyb.python.org.br/associados/guias/como-organizar-eventos/#como-solicito-a-realizacao-de-um-pagamento) há a informação que o pagamento preferencial é por boleto porque a transferência bancária é mais burocrática e precisa de cadastro prévio de quem irá receber o dinheiro.

Essa burocracia também existe para o PIX? Vi que tem essa opção no [formulário](https://www.ultradox.com/app/apyb-solicitacao-pagamento) então fiquei na dúvida se o guia estava desatualizado.

Abraço!

**Reply by cecivieira** [reply]
@vbuxbaum Não há essa burocracia no PIX

**Reply by vbuxbaum** [reply]
Obrigado!
